### DISCUSSION TITLE
Flashlight ?

### DISCUSSION THREAD

**Discussion by RubiXKube** [calibx/msfsdeck#138] [Ideas]
Why not add the flashlight (Alt+L) to the lights menu? it's really practical when you start a night flight in cold and dark
I know that adding a shortcut is very easy(i've done it), but the integration would allow to know the state of the lamp and to have the same button style

**Comment by calibx** [comment]
Hi, great idea /°

**Comment by RubiXKube** [comment]
Thanks for the other lights, but there isn't flashlight
is it a technical issue ? did you forget ? (i'm just asking)

flashlight doesn't need battery and help to find batterie button :D

**Comment by calibx** [comment]
Oh sorry, I mixed Flashlight and all lights events :(
Will be fixed in the next release if I find the event ID : https://docs.flightsimulator.com/html/Programming_Tools/Event_IDs/Event_IDs.htm

**Comment by RubiXKube** [comment]
I didin' find any event for flashlight !
i think i will keep my shortcut :(

**Comment by calibx** [comment]
I will add it as a sendKey event as there is not event for that. The only drawback is not displayed status.
